pkg/server/admin.go

```diff
@@ -719,6 +719,12 @@ func (s *adminServer) Events(
 		if err := scanner.ScanIndex(row, 4, &event.Info); err != nil {
 			return nil, err
 		}
+		if event.EventType == string(sql.EventLogSetClusterSetting) {
+			if s.getUser(req) != security.RootUser {
```

Comment:
AFAIK, this will probably always be false (`getUser` will just return `root` right now I think), so I don't think this will start redacting until session-based auth is implemented.

Reply:
Yeah, this is the kind of thing that we might forget about testing properly for UI sessions, but thanks!